command/forceleave/forceleave_test.go

```diff
@@ -56,6 +56,42 @@ func TestForceLeaveCommand(t *testing.T) {
 	})
 }
 
+func TestForceLeaveCommand_prune(t *testing.T) {
+	t.Parallel()
+	a1 := agent.NewTestAgent(t, t.Name(), ``)
```

Comment:
```suggestion
	a1 := agent.NewTestAgent(t, t.Name()+"-a1", ``)
```
Diagnostic improvement: this can help tell the logs apart if the test fails.